app/models.py

```diff
@@ -124,16 +124,19 @@ def serialize(self):
             'name': self.name,
             'slug': self.slug,
             'framework': self.framework,
-            'countersignerName': (self.framework_agreement_details or {}).get("countersignerName"),
-            'frameworkAgreementVersion': (self.framework_agreement_details or {}).get("frameworkAgreementVersion"),
-            'variations': (self.framework_agreement_details or {}).get("variations", {}),
             'status': self.status,
             'clarificationQuestionsOpen': self.clarification_questions_open,
             'lots': [lot.serialize() for lot in self.lots],
             'application_close_date': (
                 self.application_close_date and self.application_close_date.strftime(DATETIME_FORMAT)
             ),
-            'allow_declaration_reuse': self.allow_declaration_reuse
+            'allow_declaration_reuse': self.allow_declaration_reuse,
```

Comment:
I noticed this too but that's not part of this PR - think this is just a trailing comma that got added.